pfio/MultiGroupServer.F90

```diff
@@ -799,10 +806,21 @@ subroutine start_back_writers(rc)
             end select
             select type (q=>msg)
             class is (AbstractDataMessage)
+               filename =q%file_name
+               call file_timer%start()
                call thread_ptr%put_dataToFile(q, address, _RC)
+               call file_timer%stop()
             end select
             call msg_iter%next()
          enddo
+         time = file_timer%get_total()
+         file_size = file_size*4./1000./1000. ! 4-byte integer, unit is converted to MB
+         speed = file_size/time
+         write(*, "(A, F12.6, A)")"Writing "//filename//"(size ", file_size, " Mb) on node "//i_to_string(this%node_rank)
+         write(*, "(A, F12.6, A)") "time : ", time,   " s    "//filename
+         write(*, "(A, F12.6, A)") "speed: ", speed,  " Mb/s "//filename
```

Comment:
This needs to be one line of output or you need to include the process ID.  Otherwise the two lines might get a bit mixed in the log file.